incubator-tools/schema_converter_tool/schema_converter_tool.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>Remove go link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
In updated notebook, go-link removed and pointed to public documentation